service/src/main/java/bio/terra/workspace/service/resource/model/WsmCloudResourceType.java

```diff
@@ -0,0 +1,113 @@
+package bio.terra.workspace.service.resource.model;
+
+import bio.terra.common.exception.ValidationException;
+import bio.terra.workspace.generated.model.ApiResourceType;
+import javax.annotation.Nullable;
+import org.apache.commons.lang3.SerializationException;
+import org.apache.commons.lang3.StringUtils;
+
+public enum WsmCloudResourceType {
+  AI_NOTEBOOK_INSTANCE(
```

Comment:
The names here have an Azure prefix but no GCP prefix.

Reply:
I did not change these names. I just renamed the class.
I can rename that as a separate piece of work.